webapp/src/main/webapp/vue-apps/CallButton/main.js

```diff
@@ -19,9 +19,22 @@ const callPopups = new Map();
 
 export function init(settings) {
   // getting locale ressources
+  const parentContainer = document.querySelector(".leftHeaderDrawer");
+    if (parentContainer) {
+      parentContainer.addEventListener("click", e => {
+        if (e.target.classList.contains("backButton") 
+        && e.target.parentElement.classList.contains("leftHeaderDrawer")) {
```

Comment:
Better formatting would be nice here to recognize if-block clearly.

Reply:
It's formatted